datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/mutate/util/LabelUtils.java

```diff
@@ -553,10 +561,11 @@ private static GlossaryTermAssociationArray removeTermsIfExists(GlossaryTerms te
     return termAssociationArray;
   }
 
-  private static void ingestChangeProposals(List<MetadataChangeProposal> changes, EntityService entityService, Urn actor) {
+  private static void ingestChangeProposals(List<MetadataChangeProposal> changes, EntityService entityService, Urn actor, String condUpdate) {
     // TODO: Replace this with a batch ingest proposals endpoint.
+    Map<String, Long> createdOnMap = CondUpdateUtils.extractCondUpdate(condUpdate);
     for (MetadataChangeProposal change : changes) {
-      entityService.ingestProposal(change, getAuditStamp(actor), false);
+      entityService.ingestProposal(change, getAuditStamp(actor), false, createdOnMap.get(change.getEntityUrn()));
```

Comment:
I suspect `createdOnMap.get(change.getEntityUrn())` will always return `null`. Keys in `createOnMap` are String but `change.getEntityUrn()` returns a Urn. Looking at `equals` overrides, I doubt a Urn can ever be equal to a String.